base_layer/core/src/chain.rs

```diff
@@ -20,8 +20,13 @@
 // WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
 // USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
-// Portions of this file were originally copyrighted (c) 2018 The Grin Developers, issued under the Apache License,
-// Version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0.
+
+//! This file keep track of the block chain state, as well as organising the re-org etc. Because a re-org can happen, we
```

Comment:
```suggestion
//! This file provides the structs and functions that persist the block chain state, and re-org logic. Because a re-org can happen, we
```